template-tags.php

```diff
@@ -417,22 +417,58 @@ function coauthors_ids( $between = null, $betweenLast = null, $before = null, $a
 	), null, $echo );
 }
 
-function get_the_coauthor_meta( $field ) {
-	global $wp_query, $post;
+/**
+ * Outputs the co-authors Meta Data
+ *
+ * @param string $field Required The user field to retrieve.[login, email, nicename, display_name, url, type]
+ * @param string $user_id Optional The user ID for meta
+ */
+function get_the_coauthor_meta( $field, $user_id = false ) { 
+	global $wp_query, $post, $coauthors_plus;
```

Comment:
I _think_ we can safely remove the declaration of global vars `$wp_query` and `$post`, as I don't see it being used anywhere in this function.

Reply:
@rebeccahum : I have removed the unused global variables from the function as suggested by you.